src/DynamoCoreWpf/ViewModels/Core/NoteViewModel.cs

```diff
@@ -185,6 +185,7 @@ private void Select(object parameter)
         public void UpdateSizeFromView(double w, double h)
         {
             this._model.SetSize(w,h);
+            MoveNoteAbovePinnedNode();
```

Comment:
@reddyashish - this fixes the issue where a pinned note is positioned wrong when opening the graph

Reply:
Thanks @SHKnudsen.